### DISCUSSION TITLE
Cesium is not defined

### DISCUSSION THREAD

**Discussion by SarcomTD** [nshen/vite-plugin-cesium#32] [Q&A]
报错如下
![微信截图_20220728160006](https://user-images.githubusercontent.com/110159160/181453562-6ff9b0f7-ff76-4ee9-8d33-8bff6da674b5.png)
![微信截图_20220728160018](https://user-images.githubusercontent.com/110159160/181453571-333b68a1-9554-440a-a183-5a090c8d1636.png)

**Comment by nshen** [comment]
你是修改了 html 吗？ 可以试试在代码里设置

```ts
import { Viewer, Ion } from 'cesium';
import './css/main.css';

Ion.defaultAccessToken = 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa';
const viewer = new Viewer('cesiumContainer');
```

或者 
```ts
import * as Cesium from 'cesium';
Cesium.Ion.defaultAccessToken = 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaa';

```
